packages/math-input/src/components/keypad-legacy/store/types.ts

```diff
@@ -1,4 +1,4 @@
-import Key from "../../../data/keys";
```

Comment:
`Keys` is no longer exported by `.../.../.../data/keys`, but since it's a default export, that doesn't really matter for compilation purposes.